src/libraries/Common/src/System/Net/NTAuthentication.Managed.cs

```diff
@@ -296,7 +297,7 @@ internal void CloseContext()
             _serverSeal = null;
             _clientSequenceNumber = 0;
             _serverSequenceNumber = 0;
-            IsCompleted = true;
+            IsCompleted = false;
```

Comment:
why do we need this?

Reply:
`CloseContext` is basically equivalent to `Dispose` in this case. In case `IsCompleted` was `false` we should not lie that the authentication was completed. If it was `true` we throw away all the authentication information anyway, so this makes any error handling more straightforward.

Parctically speaking, `NegotiateAuthentication.RemoteIdentity` throws when authentication was not completed or when the `NegotiateAuthentication` object was disposed. It's now easier to capture both of these conditions by checking a single property.